pkgs/development/tools/icestorm/default.nix

```diff
@@ -1,23 +1,21 @@
 { stdenv, fetchFromGitHub
 , pkgconfig, libftdi
 , python3, pypy3
+, usePyPy ? stdenv.isx86_64 /* pypy3 seems broken on i686 */
```

Comment:
please document here the motivation for using pypy as a default, given that is quite uncommon.

Reply:
ok, I added a comment